src/main/scala/nl.knaw.dans.easy.solr4files/ApplicationWiring.scala

```diff
@@ -37,21 +35,12 @@ class ApplicationWiring(configuration: Configuration)
   extends DebugEnhancedLogging
     with Vault
     with Solr
-    with LdapAuthenticationComponent {
+    with AuthenticationComponent {
```

Comment:
It look s LDFAP specific, so why removing Ldap from the name?

Reply:
I melted an abstract and ldap specific variant together as there is no use for another implementation. Tried it first with a single class in download and now applied the same here. Whenever we would replace ldap with something else, we wouldn't have to rename again.

Reply:
Hm, we would still have plenty of time to rename it then. It is not like we are planning to do that anytime soon, so @jo-pol 's argument doesn't really convince me. However, the more generic name doesn't bother me much, so let's just leave it in.